debugging/netcoreapp/osProvider.ts

```diff
@@ -0,0 +1,42 @@
+/*---------------------------------------------------------
+ * Copyright (C) Microsoft Corporation. All rights reserved.
+ *--------------------------------------------------------*/
+
+import * as os from 'os';
+import * as path from 'path';
+
+export type PlatformType = 'Windows' | 'Linux';
```

Comment:
There's already an "OS" in config-utils.ts.  It should be moved out of there and made common.  Maybe rename "PlatformOS"?

Reply:
Moved out, renamed, and used from debugging implementation.